styleguide/derek/templates/landing-page/landingpage-tradeshow-event.ejs

```diff
@@ -0,0 +1,104 @@
+<div class="rsLpEvent-announcementRiver">
+  <div class="rsLpEvent-announcementRiverContainer">
+    <div class="rsLpEvent-announcementRiverRow">
+      <div class="rsLpEvent-announcementRiverTitle">
+        <h3 class="rsLpEvent-announcementRiverTitleHeader">
+          Network with local IT leaders and Rackspace SMEs. Discuss how
+          to drive business results through continuous IT modernization
+          over dinner.
+        </h3>
+      </div>
+      <div class="rsLpEvent-announcementRiverDetails">
+        <div class="rsLpEvent-announcement-col">
+          <span class="rsLpEvent-announcementRiver-secondary">
+            Matsuhisa<br/>
+            98 Steele St<br/>
+            Denver, CO<br/>
+            80206<br/>
+          </span>
+        </div>
+        <div class="rsLpEvent-announcement-col">
+          <span class="rsLpEvent-announcementRiver-secondary rsLpEvent-announcementRiver-date">
+            Thursday, April 11<br/>
+            6 - 8:30 PM
+          </span>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div>
+  <div class="event">
+    <div class="event-row">
+      <div class="event-details">
+        <div class="event-detailsTitle">Details</div>
+        <p>
+          Join us at Rackspace::Solve San Francisco, a free conference where you’ll learn how the latest cloud
+          solutions can help you scale your business, get your products to market quickly, and reduce the cost
+          of innovation.
+        </p>
+        <p>
+          Blah blah blahddy blah
+          <ul>
+            <li>I'm a list item</li>
+            <li>I'm a list item</li>
+            <li>I'm a list item</li>
+            <li>I'm a list item</li>
+            <li>I'm a list item</li>
+            <li>I'm a list item</li>
+          </ul>
+        </p>
+        <p>
+          Beyond empowering companies to tap into the power of the cloud, Rackspace also strives for social
+          empowerment. In that spirit, we’ll make a monetary donation of $15 per attendee to Girls Who Code –
+          up to $5,000. In addition to a great day of learning and networking, your attendance at Rackspace::Solve
+          will help give high school girls the skills and resources they need to pursue opportunities in computing
+          fields.
+        </p>
```

Comment:
Per Mike's mockup, can you add the CTA here? 
![image](https://user-images.githubusercontent.com/5263371/55434124-7a606280-555c-11e9-9f3c-082f7e7d9e44.png)